editors/code/rust.tmGrammar.json

```diff
@@ -600,7 +600,7 @@
 		},
 		"core_types": {
 			"comment": "Built-in/core type",
-			"name": "storage.type.core.rust",
+			"name": "entity.name.type.core.rust",
```

Comment:
This will mean that builtin types no longer get any different syntax highlighting compared to other types, right?

Reply:
Yes that’s right. I’m open to changing these choices—the main goal of this PR is just to make the TextMate grammar and the semantic colors consistent with each other, and to fix anything that’s *clearly* just a mistake/accident inherited from TextMate.